pkg/pillar/cmd/zedagent/handlevolume.go

```diff
@@ -80,11 +86,25 @@ func parseVolumeConfig(ctx *getconfigContext,
 		volumeConfig.DisplayName = cfgVolume.GetDisplayName()
 		volumeConfig.ReadOnly = cfgVolume.GetReadonly()
 		volumeConfig.RefCount = 1
+		volumeConfig.Stale = checkStaleVolume(ctx, cfgVolume, config)
 		publishVolumeConfig(ctx, *volumeConfig)
 	}
 	log.Tracef("parsing volume config done\n")
 }
 
+// checkStaleVolume returns true if there are no apps using this image in new config
```

Comment:
Hence checkVolumeHasNoAppReferences() is a better name.
s/new config/current config/